README.md

```diff
@@ -131,6 +131,11 @@ bundle exec rails server
 bundle exec rake
 ```
 
+If you just want to run a single test file you can pass the path to the file via the `SPEC` variable:
+```bash
+bundle exec rake SPEC=<path to test>
```

Comment:
This looks good.  
However, can we also add an example using rspec, which I think is a more common way to know and run individual specs.